recipe/meta.yaml

```diff
@@ -32,14 +31,13 @@ requirements:
     - s3fs ==2022.5.0
     - gcsfs ==2022.5.0
     - pyarrow ==8.0.0
-    - python-snappy ==0.6.0
     - jupyterlab ==3.4.2
     - dask-labextension ==5.2.0
     - lz4 ==4.0.0
     - ipywidgets ==7.7.0
     - numba ==0.55.1
     - scikit-learn ==1.1.1
-    - python-graphviz ==0.20
```

Comment:
I think I might still advocate for keeping `python-graphviz` in the `conda-forge` package, and then deleting it from the pip spec (while keeping `ipycytoscape` in both). Thoughts?

Reply:
I'd prefer to keep the dependencies in the `conda` and `pip` packages as close as possible. Though if the lack of `graphviz` becomes problematic, we can easily add it back in